client/src/assets/scss/styles.scss

```diff
@@ -124,6 +124,10 @@ body {
   a {
     color: $background;
   }
+  .username-button {
+    background-color: $complementary;
+    font-weight: 500;
+  }
```

Comment:
your buttons are too close together in the nav bar - add margin for separation between the elements